### PR TITLE
docs: add ngrx-handlers to resources

### PR DESCRIPTION
**Pull request by markostanimirovic**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Library is not referenced on resource page

## What is the new behavior?

Library is referenced on resource page

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/ngrx.io/content/marketing/resources.json [hunk 1]
```diff
@@ -148,6 +148,12 @@
               "url": "https://co-it.gitbook.io/ngrx-ducks/",
               "rev": true
           },
+          "ngrx-handlers": {
+            "title": "ngrx-handlers",
+            "desc": "A plugin for boilerplate reduction",
```
Comment (timdeschryver) [inline projects/ngrx.io/content/marketing/resources.json:153]: Can we think of a description without calling it boilerplate?

For example, I like how the redux toolkit describe the `createSlice` method, which is more or less what your library provides (if I'm not mistaken):

```
A function that accepts an initial state, an object full of reducer functions, and a "slice name", and automatically generates action creators and action types that correspond to the reducers and state.
```

Comment (markostanimirovic) [reply projects/ngrx.io/content/marketing/resources.json:153]: Sure 🙂

Does `A plugin for generating actions and reducer based on the handler map` sounds better?

Comment (markostanimirovic) [reply projects/ngrx.io/content/marketing/resources.json]: @timdeschryver 

I updated description with suggested text. Also, I updated the docs: https://github.com/markostanimirovic/ngrx-handlers

**Review by timdeschryver — APPROVED** [review]

File: projects/ngrx.io/content/marketing/resources.json (hunk 1 above)
Comment (timdeschryver) [reply projects/ngrx.io/content/marketing/resources.json]: Thanks @marcinmajkowski !

**Pull request merged**